mne_bids/mne_bids.py

```diff
@@ -528,32 +517,39 @@ def _sidecar_json(raw, task, manufacturer, fname, kind, eeg_ref=None,
     return fname
 
 
-def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
-                acquisition=None, run=None, kind='meg', events_data=None,
-                event_id=None, hpi=None, electrode=None, hsp=None,
-                eeg_ref=None, eeg_gnd=None, config=None,
-                overwrite=False, verbose=True):
+def write_raw_bids(raw, bids_fname, output_path, events_data=None,
+                   event_id=None, hpi=None, overwrite=False, verbose=True):
     """Walk over a folder of files and create BIDS compatible folder.
 
+    .. warning:: The original files are simply copied over. This function
+                 cannot convert data files from one format to another.
+
     Parameters
     ----------
-    subject_id : str
-        The subject name in BIDS compatible format ('01', '02', etc.)
-    task : str
-        Name of the task the data is based on.
-    raw_file : str | instance of mne.Raw
-        The raw data. If a string, it is assumed to be the path to the raw data
-        file. Otherwise it must be an instance of mne.Raw
+    raw : instance of mne.Raw
```

Comment:
for raw, you can pass the filename, correct? the docstring should be updated to reflect that

Reply:
oh no, just a `raw` instance. If you pass filename then you would also have to give all the params for the reader ...